src/resources/ha-style.ts

```diff
@@ -134,7 +134,7 @@ documentContainer.innerHTML = `<custom-style>
       --mdc-theme-primary: var(--primary-color);
       --mdc-theme-secondary: var(--accent-color);
       --mdc-theme-background: var(--primary-background-color);
-      --mdc-theme-surface: var(--card-background-color);
+      --mdc-theme-surface: var(--paper-card-background-color);
```

Comment:
We should revert this one too, it can fall back to `paper` if we really want to .